### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by remuxer32**
I have added the following changes: 

1. Highlighted important terms for better readability. Furthermore, I have fixed the disjunctive term:
 `key word` (wrong) → `keyword` (correct).

2. Highlighted `drake` which should have been highlighted before but was probably overlooked (note that the previous & following ones are highlighted).

3. Added a hyphen ( `-` ) between 32/64 and bit = **32-bit & 64-bit**. Using the hyphen is an unwritten principle in such cases; it's used almost everywhere on the Internet (see Wikipedia articles etc.).

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,10 +70,10 @@ them should be available pre-compiled for your distribution. The
...
 - A C++ compiler that supports several features of the C++11 and C++14
-  standards: initializer lists, range-based "for" loops, right angle
-  brackets, the "auto" keyword, lambda functions, the "nullptr" key
-  word, tuples, alias declarations, std::make_unique(), digit
-  separators, binary literals and generic lambdas. Others may be
+  standards: `initializer lists`, `range-based for loops`, `right angle
+  brackets`, `the auto keyword`, `lambda functions`, `the nullptr keyword`, 
+  `tuples`, `alias declarations`, `std::make_unique()`, `digit
+  separators`, `binary literals` and `generic lambdas`. Others may be
   needed, too. For GCC this means at least v4.9.x; for clang v3.4 or
```
Comment (mbunkus) [inline README.md]: You misunderstood me. `for`, `auto` etc. are keywords which deserve to be `put into backticks`. Other terms such as "lambda functions" aren't C++ keywords and shouldn't be backticked. I really meant the following *words*: `for`, `auto`, `nullptr`, `std::make_unique()`. Nothing more.

**Comment by remuxer32** [comment]
Ah, indeed, my bad. Sorry for my mistake. I had to misunderstand something. I will fix them soon.

**Comment by remuxer32** [comment]
Fixed. Now you can squash them and merge :)

**Comment by remuxer32** [comment]
A small reminder: the pull request is ready to squash and merge.

**Pull request merged**